### ISSUE TITLE
"command 'emacs-mcx.nextLine' not found"

### ISSUE THREAD

**Issue by em-exact** [whitphx/vscode-emacs-mcx#330]
Installed the extension, and getting "command 'emacs-mcx.nextLine' not found" or "command 'emacs-mcx.deleteBackwardChar' not found" - any suggestions? 

Thanks!

**Comment by tuttieee** [comment]
Thank you for reporting.
Can you show me the details of your environment?
For example, if using VSCode on macOS, you can see the summary from the menu item "Code" -> "About Visual Studio Code" then find something below
> Version: 1.45.1
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-14T08:33:47.663Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 18.7.0

In addition, a similar issue already existed here https://github.com/tuttieee/vscode-emacs-mcx/issues/136 and please check it.
This may be a cue

**Comment by em-exact** [comment]
My environment is:
```
Version: 1.46.0
Commit: a5d1cc28bb5da32ec67e86cc50f84c67cc690321
Date: 2020-06-10T08:59:06.977Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.5.0
```

In the meantime I was able to start the extension, but I'm not able to reproduce the non-working scenario any more. I've disabled en re-enabled/reloaded the extension and it seems it has helped. To reproduce, I've uninstalled the extension and did the fresh install. And the extension started working right away this time. For more context if it helps, this is with the vanilla VScode, fresh recent install on the machine.
